### ISSUE TITLE
Extramind - content embellishment

### ISSUE THREAD

**Issue by MatthewJWalls** [10xers/2015#13]
This is the weakest of the ideas I mined out of issue #6  but it's still interesting:

Feed a text document into the service, and it will automatically insert references & hyperlinks to other resources that might be useful to look at.

Use case:

For example, you might take an article on biology that you found online, and copy and paste it into the Extramind service. It will give you back the same document, but it will have marked it up to highlight technical terms and turn them into links to wikipedia pages (or other sources) that explain them. It might turn the word "Phylum" into a link to document explaining animal taxonomy. It might highlight a mathematical equation and insert a reference explaining it's construction. It could insert pictures of things that are described. It could make it so that if you hover over a technical term, it gives you a tooltip with a summary.

Basically, it embellishes an existing document with extra reference content.

**I don't think this is an idea that you could build a lasting product on**. It's too niche and people have a habit of wanting to control their output. What it WOULD do though is probably get a bunch of interest and buzz for the short term, and then you could pivot the technology elsewhere.

It's the kind of thing that would only be compelling if you get it very, very good. Just inserting hyperlinks is basically SEO-bait, the real work is doing things like dynamically including additional content and things that help you understand whatever you're reading, or give you hints on other interesting areas. Up until then, it'll be stuck in the Annoying Valley.

**Comment by MatthewJWalls** [comment]
proposed-veto because it's pretty weak.
